### DISCUSSION TITLE
GSOC 2024: Excited to meet the mentors and contribute in one of the two projects that I am interested in: Desktop Chat-Bot application OR Add Image-to-Image and Image-to-Text generation with GUI support for OpenVINO GenAI.

### DISCUSSION THREAD

**Discussion by krish1209** [openvinotoolkit/openvino#23050] [Google Summer of Code]
I Started being a part of the open source community a while back (maybe a month and a half). I have yet to make a pull request but I did participate in one of the good first issues and got assigned to it. 
I am a student in VIT Chennai, India and currently final year pursuing BTech CSE in AI ML.
Since ML and deep learning and web programming are my core fields I feel very inclined and interested to participate and contribute to one of the above mentioned projects. 
It will be a really learning and teaching journey through the world of Open-Source and thus GSOC 24.

**Comment by adrianboguszewski** [comment]
Hey @krish1209,

The mentors for Desktop Chat-Bot are @vishniakov-nikolai and @almilosz and for Image-to-image project @yangsu2022 and @FionaZZ92

**Reply by krish1209** [reply]
Thank you so much!
Do I tag them here or is there another way to contact them? 
Sorry for bothering so much 😅

**Reply by adrianboguszewski** [reply]
You can use this discussion to contact them :)

**Comment by krish1209** [comment]
Hello @vishniakov-nikolai sir and @almilosz ma'am, I would like to extend my gratitude in wanting to connect with you. 
I would like to contribute and make the project "Desktop chat bot Application" for GSOC 24. 
I feel communicating on good terms and regularly is important and I wanna follow the project with such ambition itself.

**Reply by vishniakov-nikolai** [reply]
Hello, @krish1209! Thank you for expressing your interest in implementing **Desktop Chat-Bot application**!
 
Firstly, we kindly request all participants to review the [Prerequisite task](https://github.com/openvinotoolkit/openvino/wiki/Google-Summer-Of-Code#prerequisite-task).
To begin, you may select one of the tasks listed under "good first issues" on [this page](https://github.com/orgs/openvinotoolkit/projects/3).
If you are unsure which task to choose, please feel free to send your question to us, and we will be happy to provide additional information.

**Reply by mohammedhemed77** [reply]
Hi vishniakov , 
i am passionate ML enthusiast and I am interested to join GSoC24 with openvino , 
Please help me to choose issue.

will I choose from assigned issues?

**Reply by almilosz** [reply]
Hi,
I'm happy that you are interested in OpenVINO. Apart from creating [Chat-Bot app](https://github.com/openvinotoolkit/openvino/wiki/Google-Summer-Of-Code#3-desktop-chat-bot-application) you can also prepare [a new sample for OpenVINO Node.js API](https://github.com/openvinotoolkit/openvino/wiki/Google-Summer-Of-Code#2-inference-of-neural-model-in-nodejs-environment-using-openvino).
You can choose [this issue](https://github.com/openvinotoolkit/openvino/issues/23565) to work with. Please remember to leave there a comment to be assigned and later to create a PR.

Alicja

**Comment by krish1209** [comment]
Can I start by applying and drafting a proposal to the GSOC community for OpenVINO community?

**Reply by almilosz** [reply]
Hi Krish,
I can see that you are working on this [GFI issue](https://github.com/openvinotoolkit/openvino/issues/20927).
According to the [rules](https://github.com/openvinotoolkit/openvino/wiki/Google-Summer-Of-Code#prerequisite-task) your participation in GSoC can be accepted after you create one pull request to OpenVINO repository.
I suggest creating a PR for the GFI you are working on (it doesn't have to be perfect ;) ) and applying you proposal for GSoC

Alicja

**Reply by krish1209** [reply]
Okay then I'll create a pull request as soon as possible and let you know :) 
Thank you

**Reply by krish1209** [reply]
Hi @almilosz. Hope you are doing fine. Could you please aid me in this pull request that I created. Only 2-3 changes are to be made and it is a bit unclear. Could you please help me once? After completion I would then apply for GSOC project as mentioned above.
Here is the link to the PR: https://github.com/openvinotoolkit/openvino/pull/23586

**Comment by krish1209** [comment]
Hello @almilosz ma'am. How are you doing? 
Where can I submit my GSOC OpenVINO proposal? I have it prepared and ready for review. Since you are the mentor of the project I am interested in doing, I hoped for a review of my Proposal draft from you :)
Thank you.
